app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/enums.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class BuildingName(Enum):
+    goods = "goods"
+    cotton = ("cotton",)
```

Comment:
I guess I accidentally made this a tuple somehow?

Reply:
Fixed